crates/cli/tests/reference/enums.js

```diff
@@ -83,6 +83,10 @@ export const ImplicitDiscriminant = Object.freeze({ A:0,"0":"A",B:1,"1":"B",C:42
 
 const __wbindgen_enum_ColorName = ["green", "yellow", "red"];
 
+const __wbindgen_enum_FooBar = ["foo", "bar"];
```

Comment:
This isn't used so it should not be generated as well.
I don't believe this is an issue this PR caused though.

Reply:
Yes, this is a side effect of #4147 (point 3), though I also only noticed this yesterday. I'll make a PR fixing this tomorrow. I just wanted for this PR to be in, because I'll need to implement a similar usage analysis. Nothing complex, I just wanted to see would be acceptable in the first place.